src/sections/header.html

```diff
@@ -1,4 +1,22 @@
-<div>
-    <h1 class="text-blue-800">Header</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
-</div>
\ No newline at end of file
+<section>
+    <nav class="flex justify-between p-4 mx-2 my-6 z-10 relative items-center">
```

Comment:
I'm going to talk a bit about semantics here. 
Since we're working on the header, the `header` element offers more semantic meaning in this case.

Learn more about html semantics [here](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantics_in_html). It's an important thing to keep in mind, to keep your site accessible.
```suggestion
<header>
    <nav class="flex justify-between p-4 mx-2 my-6 z-10 relative items-center">
```